terraform/domains/infrastructure/config/zones.tfvars.json

```
@@ -0,0 +1,15 @@
+{
+    "hosted_zone": {
+      "register-national-professional-qualifications.education.gov.uk": {
+        "caa_records": {},
+        "txt_records": {
+          "_dmarc": {
+            "value": "v=DMARC1;p=reject;sp=reject;adkim=s;aspf=s;fo=1;rua=mailto:dmarc-rua@dmarc.service.gov.uk,mailto:becomingateacher+dmarc@digital.education.gov.uk;ruf=mailto:dmarc-ruf@dmarc.service.gov.uk"
```

I'm not sure `becomingateacher` is correct here 🤔  On ECF we have:

```suggestion
            "value": "v=DMARC1;p=quarantine;sp=quarantine;pct=100;fo=1;rua=mailto:dmarc-rua@dmarc.service.gov.uk,mailto:dmarc-rua@education.gov.uk;ruf=mailto:DMARC.Forensic@education.gov.uk"
```

---

Updated